app/src/main/java/com/douglasbruce/tasky/features/task/form/TaskState.kt

```diff
@@ -0,0 +1,17 @@
+package com.douglasbruce.tasky.features.task.form
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import java.time.LocalDate
+import java.time.LocalTime
+
+@Parcelize
+data class TaskState(
+    val id: String = "-1",
+    val title: String? = null,
+    val description: String? = null,
```

Comment:
I'd personally also make the `id` field nullable to indicate it doesn't exist yet, since that's exactly what nulls are used for. For the title and description, you also do that, why not for the ID? 
When a project grows, it can quickly become quite confusing without proper documentation what refers to an invalid ID (and that the ID can be non-existing at all, if it's non-null)